### PR TITLE
Open source/analytics links now to be opened in a new tab

### PR DESCRIPTION
**Pull request by salochara**
👋 hey there! 

I started to play around with this project. I figured this would be a good start for contributing! 
It's a simple change, altoguh I believe it's benefitial for the application. 

Open source and open analytics links are now to be opened in a new tab so we don't take users off-site. 
CleanShot 2021-12-20 at 17 58 04@2x

Also, added .idea directory to .gitignore for people (myself included 😁) working in RubyMine

P.S. 
This would be my first contributon to OSS 🚀
I'm just starting my carreer as a RoR developer and have been following this project for a bit now. Would love to keep on contributing! 

### Pull request checklist

- [ ] Your code contains tests relevant for code you modified
- [X] You have linted and tested the project with `bin/check`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,3 +44,5 @@
 .dockerdev/*
 docker-compose.yml
 dip.yml
+
+.idea/
```
Comment (joemasilotti) [inline .gitignore:48]: Can you add a comment like line 43 that explains what this ignore is for?

**Review by joemasilotti — CHANGES_REQUESTED** [review]
This looks great, thanks so much for the contribution! I made one tiny request, other than that it is ready to merge.

File: .gitignore (hunk 1 above)
Comment (salochara) [reply .gitignore:49]: Done!

**Review by joemasilotti — APPROVED** [review]

**Comment by joemasilotti** [comment]
And merged. Thanks so much for the PR. Welcome to the community, both on railsdevs and OSS! Honored to maintain the repository of your first contribution.

**Comment by salochara** [comment]
🎉 🎉  thank you for the warm welcome!  
Honored to contribute to this project and being able to work with you!

**Pull request merged**